client/pom.xml

```diff
@@ -168,6 +168,11 @@
     </dependency>
     <dependency>
       <groupId>org.apache.cloudstack</groupId>
+      <artifactId>cloud-plugin-network-vsp</artifactId>
```

Comment:
Are nuage specific dependencies Apache 2.0 license compliant and available on maven? Only then, I would be in a favour of moving the plugin to the default build profile.